wgsl/index.bs

```diff
@@ -6055,6 +6055,8 @@ A `continue` statement must only be used in a [=statement/loop=] or [=statement/
 A `continue` statement must not be placed such that it would transfer
 control to an enclosing [=statement/continuing=] statement.
 (It is a *forward* branch when branching to a `continuing` statement.)
+i.e. generally a `continue` must not appear inside a `continuing` block. The exception being if a `loop`, `for`, or `while`
```

Comment:
Still seems only partially correct. How about:
```suggestion
Note: A `continue` can only be used in a continuing statement if it is used for transferring control flow within another loop nested in the continuing statement. That is, a `continue` cannot be used to transfer control to the start of the currently executing continuing statement.
```

Reply:
Done (though I moved it below the following sentence as I think it made more sense there as a Note)